### PR TITLE
Multi bracket validation

### PR DESCRIPTION
**Pull request by tcoots88**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [ ] Picture of whiteboard

WIll finish WB and testing

### CODE REVIEW

File: code401challenges/src/main/java/code401challenges/utilities/MultiBracketValidation.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+public class MultiBracketValidation {
+
+    public static Stack<Character> stack;
+
+
+
+    public static boolean multiBracketValidation(String input){
+        if(input.length() == 1)
+            return false;
+
```
Comment (JamesDansie) [inline code401challenges/src/main/java/code401challenges/utilities/MultiBracketValidation.java]: Need to instantiate your stack;
```suggestion
stack = new Stack<>();
```
